mne/minimum_norm/tests/test_inverse.py

```diff
@@ -381,6 +382,54 @@ def test_make_inverse_operator_free():
 
 
 @testing.requires_testing_data
+def test_make_inverse_operator_vector():
+    """Test MNE inverse computation (vector result)
+    """
+    fwd_op = read_forward_solution_meg(fname_fwd, surf_ori=True)
+    fwd_1 = read_forward_solution_meg(fname_fwd, surf_ori=False)
+    evoked = _get_evoked()
+    noise_cov = read_cov(fname_cov)
+
+    # Make different version of the inverse operator
+    inv_1 = make_inverse_operator(evoked.info, fwd_1, noise_cov, loose=1)
+    inv_2 = make_inverse_operator(evoked.info, fwd_op, noise_cov, fixed=True,
+                                  loose=None)
+    inv_3 = make_inverse_operator(evoked.info, fwd_op, noise_cov, depth=None)
+
+    # Apply the inverse operators and check the result
+    for inv in [inv_1, inv_2, inv_3]:
+        for method in ['MNE', 'dSPM', 'sLORETA']:
+            stc = apply_inverse(evoked, inv, method=method)
+            stc_vec = apply_inverse(evoked, inv, pick_ori='full',
+                                    method=method)
+            if inv['source_ori'] == FIFF.FIFFV_MNE_FIXED_ORI:
+                assert_array_almost_equal(stc.data,
+                                          stc_vec.normal(fwd_op['src']).data,
+                                          5)
```

Comment:
it is a bit unclear to me where the rounding errors come from. Mathematically, the equality should be exact. Do I use slightly different normals somehow?